draft-ietf-quic-transport.md

```diff
@@ -1626,13 +1626,16 @@ payloads of at least 1200 bytes, adding padding to packets in the datagram as
 necessary. Sending padded datagrams ensures that the server is not overly
 constrained by the amplification restriction.
 
-Packet loss, in particular loss of a Handshake packet from the server, can cause
-a situation in which the server cannot send when the client has no data to send
-and the anti-amplification limit is reached. In order to avoid this causing a
-handshake deadlock, clients MUST send a packet upon a probe timeout, as
-described in {{QUIC-RECOVERY}}. If the client has no data to retransmit and does
-not have Handshake keys, it MUST send an Initial packet in a UDP datagram of
-at least 1200 bytes.  If the client has Handshake keys, it SHOULD send a
+Loss of an Initial or Handshake packet from the server can cause a deadlock if
+the client does not send additional Initial or Handshake packets. The server can
+reach its anti-amplification limit, but the clients Initial data has been
```

Comment:
This is missing an apostrophe on Clients. But more seriously, it looks like a sentence fragment. 

Maybe move the part about the client not sending up into this sentence. 

> The server can reach its anti-amplification limit, but the client has no reason to send more packets as all the data it has sent has been acknowledged.

Reply:
Thanks for the suggestion, I took it and then tweaked it.  PTAL.